### DISCUSSION TITLE
Help with Error: GoogleJsonResponseException: API call to calendar.events.delete failed with error: Resource has been deleted

### DISCUSSION THREAD

**Discussion by SaraYBaker** [derekantrican/GAS-ICS-Sync#356] [General]
I could use some help - I'm getting the following error and I'm not sure what's happened: 

![Calendar sync error](https://github.com/derekantrican/GAS-ICS-Sync/assets/141951087/ced405b2-171c-423f-b881-a46679bba2f5)

Hoping its something straightforward someone will recognize, happy to provide details/troubleshoot as needed.  Thanks for your help.

--Sara

**Comment by Lonestarjeepin** [comment]
Did you just recently empty your Google Calendar trash?  I had this same issue late last year.  It may be completely unrelated though... I noticed the event in question has an id of "undefined" which seems odd by itself.

**Reply by SaraYBaker** [reply]
No, not that I recall...

**Comment by Lonestarjeepin** [comment]
I'm guessing it is related to the "undefined" event then which is above my pay grade.  Jonas or Derek will have to chime in on that one.  Sorry.

**Comment by AttiMM** [comment]
Hi @Lonestarjeepin , I have the same issue with the new version of script (5.8): if an event is deleted in Outlook Calendar it is not deleted in Google calendar.
![image](https://github.com/derekantrican/GAS-ICS-Sync/assets/10764890/8df3bb53-27ea-4d74-80d6-7ad150998937)
